tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -3877,3 +3894,835 @@ def test_complex128():
     assert (f_complex64() == test_complex64())
     assert (f_complex128() == test_complex128())
 
+def test_numpy_int(language):
+
+    @types('bool')
+    def test_bool_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('int')
+    def test_int_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('int8')
+    def test_int8_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('int16')
+    def test_int16_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('int32')
+    def test_int32_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('int64')
+    def test_int64_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('float')
+    def test_float_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('float32')
+    def test_float32_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
+
+    @types('float64')
+    def test_float64_int(a):
+        from numpy import int as NumpyInt
+        b = NumpyInt(a)
+        return b
```

Comment:
As discussed, could you simplify all these functions into 1 interface

Reply:
yes im working on it.